.pre-commit-hooks.yaml

```diff
@@ -22,3 +22,9 @@
   entry: buf format -w --exit-code
   types: [proto]
   pass_filenames: false
+- id: buf-mod
+  name: buf mod
+  language: golang
+  entry: buf mod
+  files: '(buf\.lock|buf\.yaml)'
```

Comment:
Why would this be for changes to buf.lock? I think it should just be for buf.yaml but I could be wrong.